source/nanoFramework.Tools.DebugLibrary.UWP/PortSerial/EventHandlerForSerialDevice.cs

```diff
@@ -153,12 +153,7 @@ private void OnAppSuspension(object sender, Windows.ApplicationModel.SuspendingE
 
                     // adjust settings for serial port
                     _device.BaudRate = 115200;
-
-                    /////////////////////////////////////////////////////////////
-                    // need to FORCE the parity setting to _NONE_ because        
-                    // the default on the current ST Link is different causing 
-                    // the communication to fail
-                    /////////////////////////////////////////////////////////////
+					_device.DataBits = 8;
```

Comment:
Please add back the comment block that was there. It seems harmless but it has cost a lot of time finding the root cause for the issue and that comment documents that. 😉